### PR TITLE
D2IQ-66671 Downgrade back to master failed

### PR DESCRIPTION
**Pull request by mediapills**
## High-level description

What features does this change enable? What bugs does this change fix?

## Corresponding DC/OS tickets (required)

  - [D2IQ-66671](https://jira.d2iq.com/browse/D2IQ-66671) Downgrade back to master failed "Cannot create a file when that file already exists: 'C:\d2iq\dcos\etc' -> 'C:\d2iq\dcos\var\tmp\etc.old'" fix.

## Related tickets (optional)

  - [D2IQ-66496](https://jira.mesosphere.com/browse/D2IQ-66496) Downgrade back to master failed.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by mediapills** [comment]
@mesosphere-mergebot request-review @jongiddy

**Comment by mediapills** [comment]
@mesosphere-mergebot request-review @sergiimatusEPAM

File: packages/winpanda/extra/src/winpanda/core/package/package.py [hunk 1]
```diff
@@ -317,6 +317,7 @@ def save_manifest(self, mheading: str=None, dpath: Path=None):
         mheading = f'{mheading}: {mhd_base}' if mheading else mhd_base
 
         try:
+            cm_utl.rmdir(str(dpath), recursive=True)
```
Comment (jongiddy) [inline packages/winpanda/extra/src/winpanda/core/package/package.py:320]: We don't need the `exist_ok=True` in the next line if we add this line, but that's a minor issue.

**Comment by jongiddy** [comment]
@mesosphere-mergebot changelog-not-required unreleased code

File: packages/winpanda/extra/src/winpanda/core/package/package.py (hunk 1 above)
Comment (mediapills) [reply packages/winpanda/extra/src/winpanda/core/package/package.py:320]: fixed

**Comment by jongiddy** [comment]
@mesosphere-mergebot changelog-not-required unreleased code

**Pull request merged**